vtysh/bgp_vty.c

```diff
@@ -3168,6 +3168,12 @@ DEFUN(neighbor_peer_group,
       "Neighbor tag\n"
       "Configure peer-group\n")
 {
+    if (strlen(argv[0]) > 80){
```

Comment:
Is there any way to define this magic '80' and use symbolic name here and whenever it used?